msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards are known for their ability to adapt to the host application's overall styling. However, their adaptability is limited when it comes to the layout. The static layout approach results in unsatisfactory user experiences, especially across different screen sizes and resolutions.
+
+Adaptive Card responsive layout allows you to design Adaptive Cards that adapt to different card widths. Responsive Adaptive Cards dynamically adjust their layout based on the actual width of the card, ensuring optimal display regardless of the context in which the card is displayed. When you design an Adaptive Card for your app, use the `targetWidth` property to make your card responsive and provide a better experience to your users on Teams mobile or desktop, regardless of the screen size.
+
+#### Configure Adaptive Card responsive layout
```

Comment:
We shouldn's use the term "configuration". Developers design or craft cards, they don't configure them. I would change this title into:

#### Design responsive Adaptive Cards